File: php/1159.php
<?php

function soma ($n){
    $cont = 0;
    for($i=0;$i<10;$i++){
        if ($n%2==0){
            $cont+=$n;
        }
        $n++;
    }
    return $cont;
}

$n = intval(fgets(STDIN));
while($n!=0){
    printf("%d\n", soma($n));
    $n = intval(fgets(STDIN));
}

?>

File: c/PROBLEMS/1159.c
#include <stdio.h>

int soma(int n){
    int cont = 0;
    for (int i =0;i<10;i++){
        if (n%2==0){
            cont+=n;
        }
        n++;
    }
    return cont;
}

int main(){
    int n, i;
    scanf("%d", &n);
    while(n!=0){
        
        printf("%d\n", soma(n));
        scanf("%d", &n);
    }
}